### ISSUE TITLE
Audio controls disabled when using tf.summary.audio

### ISSUE THREAD

**Issue by omarzouk** [tensorflow/tensorboard#2433]
I can confirm this issue both on Mac and Windows. I'm using fresh installations of Tensorboard (1.4.0) and Tensorflow (1.4.0) on both environments.

Browser: Chrome 75.0.3770.100

## Issue description

I'm trying to use `tf.summary.audio`. Upon loading tensorboard, the audio controls are shown, with the correct file duration, but the play button is disabled. I do not see any errors in my browser console, or on the Tensorboard server logs. I have tried this with `ogg` files and with other `wav` files as well.

![image](https://user-images.githubusercontent.com/18171182/61369915-76e5b280-a891-11e9-8382-2e36614e0980.png)

Here is my code

```
import os
import tensorflow as tf
from tensorflow import audio

import tensorflow as tf

with tf.Session() as sess:
	audio_raw = tf.read_file('testCropped.wav')
	
	audio_decoded = tf.contrib.ffmpeg.decode_audio(audio_raw, file_format="wav", samples_per_second=1600, channel_count=1)

	summary_audio=tf.summary.audio("test audio", tf.expand_dims(audio_decoded, 0), tf.constant(1600, dtype=tf.float32), max_outputs=1)

	writer = tf.summary.FileWriter('graphs', sess.graph)
	audio_summary_value = sess.run(summary_audio)
	
	writer.add_summary(summary=audio_summary_value)
	writer.close()
```

**Comment by nfelt** [comment]
Have you confirmed that the browser can play your audio file if opened directly in the browser?  E.g. on mac, navigating to a URL like `file:///path/to/testCropped.wav`?  If that doesn't work the issue would likely be with the audio file itself.

If that does work but TensorBoard still doesn't, is there a way you can provide the actual audio file so that the code is self-contained?  Otherwise there isn't a way to actually reproduce the behavior you see.

**Comment by omarzouk** [comment]
yes it plays fine, I can also confirm it for `ogg`. I can't distribute that file in particular due to licensing, but I have also tested my code with [this file](https://file-examples.com/wp-content/uploads/2017/11/file_example_WAV_1MG.wav) and I can confirm that it reproduces the problem. The new file is also playable in my browser normally.

**Comment by omarzouk** [comment]
Ok, so turns out it works on Microsoft Edge browser....

I noticed that the audio was bad, and the reason was that I had a low samples per second because I missed a 0 `samples_per_second=1600`. Upon changing this to 16,000, it started working on Chrome again.

So apparently Chrome doesn't play audio if it has too few samples per second.

**Comment by nfelt** [comment]
Glad you were able to figure out the issue, thanks for following up.
